app/mailers/lesson_mailer.rb

```diff
@@ -16,6 +16,7 @@ def lesson_email(user:, course:, lesson:, languages:, disguised:)
               end
 
     LocalesService.with(@languages.first) do
+      attachments.inline['icon.png'] = URI.parse(@course.image).read
```

Comment:
Maybe for later; is it worth caching these images locally (e.g. in Redis cache)?